napalm/nxos/nxos.py

```diff
@@ -982,7 +1003,7 @@ def get_ntp_stats(self):
 
     def get_interfaces_ip(self):
         interfaces_ip = {}
-        ipv4_command = 'show ip interface'
+        ipv4_command = 'show ip interface vrf all'
```

Comment:
Why is this changing? That probably should be in a separate change if it needs to change?

Reply:
I was adding the vrf support at that time, and yes it's not related to the bgp_neighbors